### PR TITLE
feat(Th): add table header sortable props

### PR DESCRIPTION
**Pull request by tujoworker**

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/components/table/Examples.js [hunk 1]
```diff
@@ -120,36 +120,36 @@ export const TableLongHeader = () => (
 <Table>
   <caption className="dnb-sr-only">A Table Caption</caption>
   <thead>
-    <tr className="dnb-table--small">
-      <th scope="col" colSpan="2">
+    <Tr className="dnb-table--small">
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/components/table/Examples.js:123]: Should this example, `TableLongHeader`, still use `className` for the `Tr` and `Th` components?
Or should the usage of className rather be represented by the usage of the component's properties instead?

File: packages/dnb-design-system-portal/src/docs/uilib/components/table/Examples.js [hunk 2]
```diff
@@ -180,61 +180,61 @@ export const TableSticky = () => (
...
   <thead>
-    <tr>
-      <th scope="col" colSpan="2">
+    <Tr>
+      <Th scope="col" colSpan="2">
         Header
-      </th>
-      <th scope="col" className="dnb-table--sortable dnb-table--reversed">
+      </Th>
+      <Th scope="col" className="dnb-table--sortable dnb-table--reversed">
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/components/table/Examples.js:187]: Should this example, `TableSticky`, still use `className` for the `Th` components?
Or should the usage of className rather be represented by the usage of the component's properties instead?

File: packages/dnb-eufemia/src/components/table/TableTd.tsx [hunk 3]
```diff
@@ -7,12 +7,6 @@ export type TableTdProps = {
    * Default: null
    */
   children?: React.ReactNode
-
-  /**
-   * Custom className on the component root
-   * Default: null
-   */
-  className?: string
```
Comment (langz) [inline packages/dnb-eufemia/src/components/table/TableTd.tsx:15]: As the prop `className` is removed, should we also remove the usage of it in this component?

File: packages/dnb-eufemia/src/components/table/TableTh.tsx [hunk 4]
```diff
@@ -8,10 +8,28 @@ export type TableThProps = {
   children: React.ReactNode
 
   /**
-   * Custom className on the component root
-   * Default: null
+   * Defines the table header as sortable
+   * Default: false
    */
-  className?: string
```
Comment (langz) [inline packages/dnb-eufemia/src/components/table/TableTh.tsx:14]: As the prop `className` is removed, should we also remove the usage of it in this component?

File: packages/dnb-eufemia/src/components/table/TableTr.tsx [hunk 5]
```diff
@@ -15,12 +15,6 @@ export type TableTrProps = {
   children:
     | React.ReactElement<TableTdProps>
     | Array<React.ReactElement<TableTdProps>>
-
-  /**
-   * Custom className on the component root
-   * Default: null
-   */
-  className?: string
```
Comment (langz) [inline packages/dnb-eufemia/src/components/table/TableTr.tsx:23]: As the prop `className` is removed, should we also remove the usage of it in this component?

**Review by langz — COMMENTED** [review]
Looks good 🙏 

The main thing I didn't completely get is why we remove the prop `className` from all components Td, Th, Tr, but that we don't remove the usage of it? 🤔 
Is it because we only want to use it internally, and not "publicly promote it"? 🤔

File: packages/dnb-eufemia/src/components/table/TableTh.tsx (hunk 4 above)
Comment (tujoworker) [reply packages/dnb-eufemia/src/components/table/TableTh.tsx:14]: `className` is still supported, but via the `HTMLTableCellElement` 👍

File: packages/dnb-eufemia/src/components/table/TableTr.tsx (hunk 5 above)
Comment (tujoworker) [reply packages/dnb-eufemia/src/components/table/TableTr.tsx:23]: `className` is still supported, but via the `HTMLTableRowElement` 👍

File: packages/dnb-design-system-portal/src/docs/uilib/components/table/Examples.js (hunk 1 above)
Comment (tujoworker) [reply packages/dnb-design-system-portal/src/docs/uilib/components/table/Examples.js:123]: Yes, I will remove this one in another PR 👍

File: packages/dnb-design-system-portal/src/docs/uilib/components/table/Examples.js (hunk 2 above)
Comment (tujoworker) [reply packages/dnb-design-system-portal/src/docs/uilib/components/table/Examples.js:187]: This will also be solved in a different PR 😁

**Comment by tujoworker** [comment]
> The main thing I didn't completely get is why we remove the prop `className` from all components Td, Th, Tr, but that we don't remove the usage of it? 🤔 Is it because we only want to use it internally, and not "publicly promote it"? 🤔

Eufemia started with only table styles. But now that we have a component, we should go away from supporting "all features" with classes as well. This way we have less documentation to maintain.

**Review by langz — APPROVED** [review]
Thanks for looking into my comments, I think I understand it all now 🧠 💪

File: packages/dnb-eufemia/src/components/table/TableTr.tsx (hunk 5 above)
Comment (langz) [reply packages/dnb-eufemia/src/components/table/TableTr.tsx:23]: Of course, my bad 😹

**Pull request merged**